pkg/visibility/api/rest/pending_workload_CQ_test.go

```diff
@@ -270,6 +270,10 @@ func TestPendingWorkloads(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			manager := queue.NewManager(utiltesting.NewFakeClient(), nil)
 			ctx := context.Background()
+			ctx, cancel := context.WithTimeout(ctx, 100*time.Millisecond)
```

Comment:
This timeout seems very short, I'm worried it might fail on loaded environment, It should not hurt to increase the timeout, to say 5s.